src/artifact.rs

```diff
@@ -278,12 +286,22 @@ impl Artifact {
     /// **NB**: If either `link.from` or `link.to` is undeclared, then this will return an error.
     /// If `link.from` is an import you previously declared, this will also return an error.
     pub fn link<'a>(&mut self, link: Link<'a>) -> Result<(), Error> {
+        self.link_aux(link, None)
+    }
+    /// A variant of `link` with a RelocOverride provided. Has all of the same invariants as
```

Comment:
If we keep (probably will) this we need to add note that this is kind of really unsafe unless you know exactly what you’re doing ;)